index.html

```diff
@@ -935,8 +951,11 @@ <h2>
           <li>
             <p>
               Otherwise, present a user interface to allow the user to interact
-              with the <var>handlers</var>. The user agent SHOULD prioritize
-              the preference of the user when presenting payment methods.
+              with the <var>handlers</var>. Optionally, if
+              <var>request</var>.<a>[[\uiLang]]</a> is not null, localize the
+              user interface to match, as closely as possible, the language of
+              <a>[[\uiLang]]</a>. The user agent SHOULD prioritize the
```

Comment:
I think you should paragraph break before the last sentence as now it's somewhat disconnected from the rest.